theme/templates/resource-landing-page/citation.html

```diff
@@ -216,15 +216,6 @@
             {% endif %}
         </div>
         {% if show_manage_access %}
-            <a type="button" class="btn btn-default" data-toggle="modal" data-target="#manage-access">
-                    <span class="glyphicon glyphicon-lock"><span
-                            class="button-label"> Manage access</span></span>
-            </a>
-
-            <span data-toggle="tooltip" data-placement="auto"
-                  title='Use this button to share your resource with specific HydroShare users. You can give other users the ability to view or edit this resource. You can also add additional owners who will have full permissions.'
-                  class="glyphicon glyphicon-info-sign text-muted"></span>
-
             {#======= Manage access modal window begins =======#}
             <div class="modal fade" id="manage-access">
                 <div class="modal-dialog">
```

Comment:
@mauriel133 Would it be better to move this manage-access modal dialog into modals.html together with other modal dialogs now that manage_access button is moved out of citation area on landing page?